pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -335,6 +337,13 @@ func Run() {
 	flowStatTimer := flextimer.NewRangeTicker(time.Duration(fmin),
 		time.Duration(fmax))
 
+	nexthopProbeIntv := time.Duration(NhProbeInteval * time.Second) // 15 sec
+	pmax := float64(nexthopProbeIntv)
+	pmin := pmax * 0.9
+	HostProbeTimer := flextimer.NewRangeTicker(time.Duration(pmin),
+		time.Duration(pmax))
+	zedrouterCtx.checkNIUplinks = make(chan bool)
```

Comment:
Sure you want the sender to block until the receiver comes to receive on the channel? Might make more sense to have a depth of 1 for the channel.

Reply:
will look into the this depth